### PR TITLE
DSA | Updated 105, Sorting Algorithm links

### PR DESCRIPTION
**Pull request by ysmnikhil**
This MR update links on 105
- Quick Sort
- Merge Sort
- Bubble Sort

### CODE REVIEW

File: src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # Merge Sort
 
 __Merge sort__ is a type of sorting algorithm that follows the divide-and-conquer paradigm. It was invented by John von Neumann in 1945. This algorithm works by dividing an unsorted list into `n` partitions, each containing one element (a list of one element is considered sorted), then repeatedly merging partitions to produce new sorted lists until there is only 1 sorted list remaining. This resulting list is the fully sorted list. The process of dividing the list is done recursively until it hits the base case of a list with one item. Merge sort has a time complexity of `O(n log n)` for all cases (best, average and worst), which makes it highly efficient for large data sets.
+
+Learn more from the following resources:
+
+- [@video@Merge Sort](https://www.youtube.com/watch?v=p__ETf2CKY4)
+- [@article@Merge Sort Visulize](https://www.hackerearth.com/practice/algorithms/sorting/merge-sort/visualize/)
```
Comment (dansholds) [inline src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md:8]:
```suggestion
Learn more from the following resources:

- [@video@Merge Sort](https://www.youtube.com/watch?v=4VqmGXwpLqc)
- [@article@Merge Sort Visulize](https://www.hackerearth.com/practice/algorithms/sorting/merge-sort/visualize/)
```

Comment (dansholds) [inline src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md:8]:
```suggestion
Learn more from the following resources:

- [@video@Merge Sort](https://www.youtube.com/watch?v=4VqmGXwpLqc)
- [@article@Merge Sort Visulize](https://www.hackerearth.com/practice/algorithms/sorting/merge-sort/visualize/)
```

Comment (dansholds) [reply src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md:8]: Placed with a Merge sort video as you included the same video as in Bubble sort.

**Comment by dansholds** [comment]
@ysmnikhil awesome thanks, love the hackerearth visualisations!

File: src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md (hunk 1 above)
Comment (ysmnikhil) [reply src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md]: Thanks for the Update.

The video which I added that was nice, but I was unable to find that in his own YouTube channel, so I added this one. But I will keep this in mind to add only from valid channels.

Comment (dansholds) [reply src/data/roadmaps/datastructures-and-algorithms/content/105-sorting-algorithms/101-merge-sort.md]: I liked the video, you just added his bubble sort video to both the bubble sort topic and the merge sort one!

**Pull request merged**